libs/features/seotech/seotech.js

```diff
@@ -0,0 +1,34 @@
+export const SEOTECH_API_URL_PROD = 'https://14257-seotech.adobeioruntime.net';
+export const SEOTECH_API_URL_STAGE = 'https://14257-seotech-stage.adobeioruntime.net';
+
+export function logError(msg) {
+  window.lana?.log(`SEOTECH: ${msg}`, {
+    debug: false,
+    implicitSampleRate: 100,
+    sampleRate: 100,
+    tags: 'errorType=seotech',
+  });
+}
+
+export async function getVideoObject(url, seotechAPIUrl) {
+  const videoUrl = new URL(url)?.href;
```

Comment:
Although this is called from within a `try/catch` block and it definitely works, if you move some things around in the future and this gets taken out of the `try/catch` block, calling this method will throw errors if `url` is not defined or is invalid. It's definitely a matter or preference, but I usually go for checking the issue where it might occur.

Reply:
If that happens then the related test will fail (see L18 in seotech.test.js), but more to your point, the test will fail because there is a test to begin with -- it's easy to forget to write it since coverage tools count try/catch once, even though it wraps two statements that will branch. I'm not the biggest exception fan either for this reason, but it was useful here.